GitUI/GitUICommands.cs

```diff
@@ -2421,11 +2420,6 @@ internal void RaisePostRegisterPlugin(IWin32Window owner)
             InvokeEvent(owner, PostRegisterPlugin);
         }
 
-        public void BrowseGoToRef(string refName, bool showNoRevisionMsg)
```

Comment:
This will be needed here: https://github.com/gitextensions/gitextensions/pull/4499/files#diff-d2289866a7ce1fef49a06a13dfb64391R97

Reply:
Thanks. Marked WIP for now and will revisit once #4499 is merged.